examples/docking-protein-DNA/data/run.cns.patch

```diff
@@ -0,0 +1,74 @@
+--- run.cns	2019-03-11 12:07:52.402707184 +0100
```

Comment:
Why there is a CNS file in the examples folder?

Reply:
oh, forgot to remove this file